aspnetcore/fundamentals/configuration/index.md

```diff
@@ -1,441 +1,365 @@
 
-## Default configuration
+1. *appsettings.json*
+1. *appsettings.*`Environment`*.json* : For example, the *appsettings*.***Production***.*json* and *appsettings*.***Development***.*json* files. The environment version of the file is loaded based on the [IHostingEnvironment.EnvironmentName](xref:Microsoft.Extensions.Hosting.IHostingEnvironment.EnvironmentName*). For more information, see <xref:fundamentals/environments>.
 
-Web apps based on the ASP.NET Core [dotnet new](/dotnet/core/tools/dotnet-new) templates call <xref:Microsoft.Extensions.Hosting.Host.CreateDefaultBuilder*> when building a host. `CreateDefaultBuilder` provides default configuration for the app in the following order:
+*appsettings*.`Environment`.*json* values override keys in *appsettings.json*. For example, by default:
 
-The following applies to apps using the [Generic Host](xref:fundamentals/host/generic-host). For details on the default configuration when using the [Web Host](xref:fundamentals/host/web-host), see the [ASP.NET Core 2.2 version of this topic](/aspnet/core/fundamentals/configuration/?view=aspnetcore-2.2).
+* In development, *appsettings*.***Development***.*json* configuration overwrites values found in *appsettings.json*.
+* In production, *appsettings*.***Production***.*json* configuration overwrites values found in *appsettings.json*. For example, when deploying the app to Azure.
 
-* Host configuration is provided from:
-  * Environment variables prefixed with `DOTNET_` (for example, `DOTNET_ENVIRONMENT`) using the [Environment Variables Configuration Provider](#environment-variables-configuration-provider). The prefix (`DOTNET_`) is stripped when the configuration key-value pairs are loaded.
-  * Command-line arguments using the [Command-line Configuration Provider](#command-line-configuration-provider).
-* Web Host default configuration is established (`ConfigureWebHostDefaults`):
-  * Kestrel is used as the web server and configured using the app's configuration providers.
-  * Add Host Filtering Middleware.
-  * Add Forwarded Headers Middleware if the `ASPNETCORE_FORWARDEDHEADERS_ENABLED` environment variable is set to `true`.
-  * Enable IIS integration.
-* App configuration is provided from:
-  * *appsettings.json* using the [File Configuration Provider](#file-configuration-provider).
-  * *appsettings.{Environment}.json* using the [File Configuration Provider](#file-configuration-provider).
-  * [Secret Manager](xref:security/app-secrets) when the app runs in the `Development` environment using the entry assembly.
-  * Environment variables using the [Environment Variables Configuration Provider](#environment-variables-configuration-provider).
-  * Command-line arguments using the [Command-line Configuration Provider](#command-line-configuration-provider).
+<a name="optpat"></a>
 
-## Security
+#### Bind hierarchical configuration data using the options pattern
```

Comment:
It would probably fit better there, but I want to show it right away. Most of the time you should be using the options pattern. I don't want folks to read enough to get *appsettings.json* working then quit reading the doc and miss this.